wled00/data/cpal/cpal.htm

```diff
@@ -520,8 +521,10 @@ <h1 style="display: flex; align-items: center;">
     if (hst.length > 0 ) {
       try {
         var arr = [];
-        const response = await fetch('http://'+hst+'/json/info');
-        const json = await response.json();
+        const responseInfo = await fetch('http://'+hst+'/json/info');
```

Comment:
Perhaps also use `getURL()` logic from `index.js` which will work behind reverse proxy.

Reply:
I did it like it was done before. But it you would prefer it we could also use `getURL()`.

Reply:
The only problem I see is that if we do this we have to copy the code from index.js to cpal.htm and then we have created duplicate code again. We could wait with this until we have a `global.js` file that contains functions like `getURL()` that are used on every page, so we do not have to maintain the duplicate code. But I'll let you decide. Should I just copy the code now, or should we wait until we have a `global.js` file?